Documentation/ColumnsConfig/Type/Flex/AboutDataStructures.rst

```diff
@@ -85,6 +85,10 @@ on a selected "tt_content" element plugin.
 Data structure lookup with tree traversal
 =========================================
 
+..  note::
```

Comment:
I would suggest to just remove the example in v13?

Reply:
The PR is for 12.4 (see "Releases" line), just hinting: It will be gone in v13 but you can already migrate.

Reply:
Woops sorry, missed that part